### DISCUSSION TITLE
Audio Mixer does not have a bind in settings

### DISCUSSION THREAD

**Discussion by rasikhq** [ppy/osu#18367] [Q&A]
Is there an option to change Audio Mixer's bind?

CTRL+F9 brings it up but this cannot be changed, at-least I couldn't find an option. My recording start/stop is CTRL+F9

I did a quick search and can't find any issue/discussion about it, sorry if I missed it. Would be nice to be able to change the bind.

Thanks!

**Comment by peppy** [comment]
That is a developer tool and cannot be changed.

Can you explain why you would be looking to change it?

**Reply by rasikhq** [reply]
Like I mentioned that I have my start/stop recording hotkey on CTRL+F9.

If it cannot be changed then its fine I'll just use a different keybind for osu to start/stop recording clips with OBS. However there could be a section in settings (I believe there is already) where we should be able to turn off developer tools if we can't re-bind them atleast.

**Reply by peppy** [reply]
OBS should be consuming the hotkey. It's weird that it also reaches the game. Eventually we may default these to off in release, we'll see.
